lib/iris/tests/unit/cube/test_Cube__operators.py

```diff
@@ -160,5 +161,26 @@ def test_div_cubes(self):
         self.assertTrue(isinstance(data, np.ndarray))
         self.assertEqual(data.shape, ())
 
+
+class Test_Masked_Lazy_Maths(tests.IrisTest):
+
+    def build_lazy_cube(self):
```

Comment:
Indeed, I agree. All 3 in this file are different though so I'm not going to implement anything in this PR.